javascript/linked-list/insertions.md

```diff
@@ -0,0 +1,20 @@
+# Linked List Implementation
+<!-- Description of the challenge -->
+
+## Challenge
+
+Extending an Implementation
+
+### Features
+
+- Linked List
+- Node
+- Append
+- Insert before
+- Insert after
```

Comment:
In the future I would like a description of the methods here

Reply:
Will do it in the future! Sorry just saw this comment. Will explain the methods in today's challenge.